Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM quay.io/fedora/fedora:33-x86_64
```

Comment:
could we use much more smaller images there ?

Reply:
Built on purpose on fedora to reduce the delta between container and local build. As we run 1 image instead of 4, I believe benefit is still there.

Reply:
For dogfooding I don't still see the benefits ? it looks like more a "I don't use che" use-case ?

Reply:
> Built on purpose on fedora to reduce the delta between container and local build. As we run 1 image instead of 4, I believe benefit is still there.

We're trying to have multi-arch images as well on che, here it seems it's not multi-arch then

Reply:
These are the constraints I saw when choosing a base image:
* Contains glibc (for vale) -> rules out alpine.
* As I understood, the license of ubi8 makes it unsuitable for this project?
* Maximize the number of available system packages:  newdoc is available on fedora
* No system package (on fedora) for linkchecker, yq, antora, asciidoctor.js, gulp, vale.
* To build on Quay without rate-limiting issue: base image should not be published on Docker Hub -> rules out the fedora images published on Docker Hub https://hub.docker.com/_/fedora.
* Fedora is also present on quay: https://quay.io/repository/fedora/fedora?tag=latest&tab=tags: multiarch in only for F34. I opted for stability (F33, but not multiarch). Of course we can discuss this choice. Is multiarch really a blocker?

Reply:
> Contains glibc (for vale) -> rules out alpine.

Official image of vale is using Alpine : https://github.com/errata-ai/vale/blob/v2/Dockerfile

license of ubi8 is fine, but ubi8 is still bigger

Reply:
Switched to alpine as baseimage (approx. 200M instead of 400M).
    Building vale and newdoc to have multiarch binaries.